tests/utils/index.js

```diff
@@ -1250,6 +1251,18 @@ const logFeedbackDocs = async (test) => {
   return true;
 };
 
+const getMinimumChromeVersion = () => {
+  return MINIMUM_BROWSER_VERSION;
+};
+
+const isMinimumChromeVersion = () => {
+  // eslint-disable-next-line no-undef
+  if (driver.capabilities.browserVersion.split('.').shift() === MINIMUM_BROWSER_VERSION) {
```

Comment:
There's a little complexity here with all the different browser versions... we have:

- `driver.capabilities.browserVersion`
- `MINIMUM_BROWSER_VERSION`
- `process.env.CHROME_VERSION`

I think it would be easier if we just don't use `driver.capabilities.browserVersion` if we can, for example, use the env variable here too, eg:

```suggestion
  if (process.env.CHROME_VERSION === MINIMUM_BROWSER_VERSION) {
```

That way it's more consistent throughout the code.

This also means throughout the `tests/e2e/wdio.conf.js` file you can use `isMinimumChromeVersion` rather than `getMinimumChromeVersion` which will simplify the code there a little too.

This might mean you can remove the `getMinimumChromeVersion` function altogether...

What do you think?